src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -655,16 +657,25 @@ private void AddServicesProducedInfo(IResource modelResource, IAnnotationHolder
 
     private async Task CreateResourcesAsync<RT>(CancellationToken cancellationToken) where RT : CustomResource
     {
-        var resourcesToCreate = _appResources.Select(r => r.DcpResource).OfType<RT>();
-        if (!resourcesToCreate.Any())
+        try
         {
-            return;
-        }
+            var resourcesToCreate = _appResources.Select(r => r.DcpResource).OfType<RT>();
+            if (!resourcesToCreate.Any())
+            {
+                return;
+            }
 
-        // CONSIDER batched creation
-        foreach (var res in resourcesToCreate)
+            // CONSIDER batched creation
+            foreach (var res in resourcesToCreate)
+            {
+                await kubernetesService.CreateAsync(res, cancellationToken).ConfigureAwait(false);
+            }
+        }
+        catch (OperationCanceledException ex)
         {
-            await kubernetesService.CreateAsync(res, cancellationToken).ConfigureAwait(false);
+            // We catch and suppress the OperationCancelledException because the user may CTRL-C
+            // during start up of the resources.
+            _logger.LogInformation("Cancellation during creation of resources detected: {ex}", ex);
```

Comment:
Why log at all?

Reply:
We actually suppress this log category by default but I thought it might be helpful for diagnostics if we ask people to turn those log categories on.